src/renderer/screens/manager/AppsList/AppActions.js

```diff
@@ -169,27 +222,26 @@ const AppActions: React$ComponentType<Props> = React.memo(
                     </Box>
                   </Button>
                 </Tooltip>
-              ) : (
+              ) : hasSpecificAction ? (
                 <Button
-                  color={canAddAccount ? "palette.primary.main" : "palette.text.shade40"}
-                  inverted
+                  color={"palette.primary.main"}
                   style={{ display: "flex", backgroundColor: "rgba(0,0,0,0)" }}
                   fontSize={3}
-                  onClick={onNavigateToPlatform}
+                  onClick={() => onNavigateTo(type)}
```

Comment:
Could this this simply be `onClick={onNavigateTo}` with `onNavigateTo` being declared like this and using `type` directly instead of as an argument ?
```ts
const onNavigateTo = useCallback(() => {
    /* here do what you have to do and use `app.type` directly*/
}, [app.name, history, app.type]);
```
This arrow func declared in the props will create unnecessary renders no ?

Reply:
You are right :)